fdbrpc/include/fdbrpc/LoadBalance.actor.h

```diff
@@ -250,7 +249,8 @@ Future<Void> replicaComparison(Req req,
                                Future<ErrorOr<Resp>> fSource,
                                uint64_t srcEndpointId,
                                Reference<MultiInterface<Multi>> ssTeam,
-                               RequestStream<Req, P> Interface::*channel) {
+                               RequestStream<Req, P> Interface::*channel,
+                               int requiredReplicas) {
```

Comment:
Can you add a description for this function, including how `requiredReplicas` changes behavior?

Reply:
Done.